### ISSUE TITLE
Alsa module goes up in a percentage of 4, even though it's 5

### ISSUE THREAD

**Issue by JimPix1** [polybar/polybar#2399]
## Checklist

* [x] I have read the appropriate section in the [contributing
  guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
* [x] I believe this issue is a problem with polybar itself and not a misconfiguration on my part.
* [x] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that
  may have already reported this problem.
* [x] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.

## Describe the bug

When I press my volume up keybind, it goes up by 5, but polybar says it goes up in 4, e.g i press it once to get to 5 percent, polybar says it's 4 percent instead. This happens with any number, polybar will always be one percent lower.
**Expected behavior:**

I expected polybar to display 5 percent as 5 percent instead of 4.

**Actual behavior:**

Polybar displays it as 4 instead of 5, 30% is 29% etc, occurs with all numbers.

**Was it working before?**
* Did you also experience this bug in an earlier version of polybar (yes/no/don't know)?
* If no, what was the last version where this worked correctly?
this never worked properly

## To Reproduce
use the alsa module with <label-volume> and increase the volume by any number, polybar will always have the number be one percent lower

a standard config with the alsa/volume module.

Change the volume either with a keybind or a command such as pactl set-sink-volume
## Environment:
* WM: i3
* Distro: ArcoLinux
* Output of `polybar -vvv`:
```
polybar 3.5.4

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /usr/bin/c++
Compiler flags: -D_FORTIFY_SOURCE=2 -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now
```

**Comment by patrick96** [comment]
What exact commands are you using to change the volume?

Against what are you comparing the volume displayed by polybar? The volume on the bar should match the volume displayed by the `amixer` command (depending on how the module is configured).

Can you please also post the alsa module config that you are using because I don't know what you mean by "a standard config with the alsa/volume module". There is a sample config that contains an alsa module, but that module doesn't display percentages (just a progress bar).

**Comment by JimPix1** [comment]
I'm comparing it with pavucontrol, and I looked at amixer and it's still the same, polybar is off by one percent.
heres the config
```[module/alsa]
type = internal/alsa

; Soundcard to be used
; Usually in the format hw:# where # is the card number
; You can find the different card numbers in `/proc/asound/cards`
master-soundcard = default
speaker-soundcard = default
headphone-soundcard = default

; Name of the master, speaker and headphone mixers
; Use the following command to list available mixer controls:
; $ amixer scontrols | sed -nr "s/.*'([[:alnum:]]+)'.*/\1/p"
; If master, speaker or headphone-soundcard isn't the default, 
; use `amixer -c # scontrols` instead where # is the number 
; of the master, speaker or headphone soundcard respectively
;
; Default: Master
master-mixer = Master

; Optionally define speaker and headphone mixers
; Default: none
;;speaker-mixer = Speaker
; Default: none
;;headphone-mixer = Headphone

; NOTE: This is required if headphone_mixer is defined
; Use the following command to list available device controls
; $ amixer controls | sed -r "/CARD/\!d; s/.*=([0-9]+).*name='([^']+)'.*/printf '%3.0f: %s\n' '\1' '\2'/e" | sort
; You may also need to use `amixer -c # controls` as above for the mixer names
; Default: none
;;headphone-id = 9

; Use volume mapping (similar to amixer -M and alsamixer), where the increase in volume is linear to the ear
; Default: false
;;mapped = true

; Interval for volume increase/decrease (in percent points)
; Default: 5
interval = 5

; Available tags:
;   <label-volume> (default)
;   <ramp-volume>
;   <bar-volume>
format-volume = <ramp-volume> <label-volume>

; Available tags:
;   <label-muted> (default)
;   <ramp-volume>
;   <bar-volume>
format-muted = <label-muted>
format-muted-prefix = 
format-muted-prefix-foreground = ${color.red}

; Available tokens:
;   %percentage% (default)
label-volume = %percentage%%

; Available tokens:
;   %percentage% (default
label-muted = " Muted"
label-muted-foreground = ${color.sep}

; Only applies if <ramp-volume> is used
ramp-volume-0 = 
ramp-volume-1 = 
ramp-volume-2 = 
ramp-volume-foreground = ${color.blue}

; If defined, it will replace <ramp-volume> when
; headphones are plugged in to `headphone_control_numid`
; If undefined, <ramp-volume> will be used for both
; Only applies if <ramp-volume> is used
ramp-headphones-0 = 
ramp-headphones-background = ${color.blue}
```

**Comment by patrick96** [comment]
Fixed in #2401, this will be in the 3.6.0 release
